### ISSUE TITLE
test_obj_isa is failing

### ISSUE THREAD

**Issue by opoplawski** [gnudatalanguage/gdl#151]
``
142: Test command: /export/home/orion/fedora/gdl/gdl-0.9.8/build/testsuite/launchtest "test_obj_isa.pro"
142: Test timeout computed to be: 9.99988e+06
142: % Compiled module: TEST_OBJ_ISA.
142: % TEST_OBJ_ISA: Errors in Testing LIST
142: % TEST_OBJ_ISA: Errors in Testing OBJARR
142: % Compiled module: BANNER_FOR_TESTSUITE.
142: % Compiled module: GDL_IDL_FL.
142: % TEST_OBJ_ISA: ====================================================
142: % TEST_OBJ_ISA: =                                                  =
142: % TEST_OBJ_ISA: =  2 errors encountered during TEST_OBJ_ISA tests  =
142: % TEST_OBJ_ISA: =                                                  =
142: % TEST_OBJ_ISA: ====================================================
 8/13 Test # 142: test_obj_isa.pro .................***Failed    0.22 sec
```

**Comment by maynardGK** [comment]
This won't be an issue once upcoming patches are implemented. Below, I run 0.9.8 SVN (where failure
will occur) and my own code which has some (few) "GDL internals" patches and an extensive LIST rewrite.

 Probably, the errors will go away before LIST gets a change.

`$ gdlsvn

  GDL - GNU Data Language, Version 0.9.8 svn

- For basic information type HELP,/INFO
- Please report bugs, feature or help requests and patches at:
  http://sourceforge.net/projects/gnudatalanguage/

% Compiled module: PATH_SEP.
GDL> help,!gdl
** Structure !GNUDATALANGUAGE, 3 tags, length=80, data length=24:
   RELEASE         STRING    '0.9.8 svn'
   BUILD_DATE      STRING    'Apr 10 2018'
   EPOCH           LONG        1523343600
GDL> test_obj_isa
% Compiled module: TEST_OBJ_ISA.
% TEST_OBJ_ISA: Errors in Testing LIST
% TEST_OBJ_ISA: Errors in Testing OBJARR
% Compiled module: BANNER_FOR_TESTSUITE.
% Compiled module: GDL_IDL_FL.
% TEST_OBJ_ISA: ====================================================
% TEST_OBJ_ISA: =                                                  =
% TEST_OBJ_ISA: =  2 errors encountered during TEST_OBJ_ISA tests  =
% TEST_OBJ_ISA: =                                                  =
% TEST_OBJ_ISA: ====================================================

greg@Homerw7  /d/home/greg/testing
$ gdlf

  GDL - GNU Data Language, Version 0.9.7 (/f/gdl )

- For basic information type HELP,/INFO
- Please report bugs, feature or help requests and patches at:
  http://sourceforge.net/projects/gnudatalanguage/

% Compiled module: PATH_SEP.
GDL> test_obj_isa
% Compiled module: TEST_OBJ_ISA.
% Compiled module: BANNER_FOR_TESTSUITE.
% Compiled module: GDL_IDL_FL.
% TEST_OBJ_ISA: =====================================================
% TEST_OBJ_ISA: =                                                   =
% TEST_OBJ_ISA: =  NO errors encountered during TEST_OBJ_ISA tests  =
% TEST_OBJ_ISA: =                                                   =
% TEST_OBJ_ISA: =====================================================
GDL> help,/sour
Compiled Procedures:
$MAIN$
BANNER_FOR_TESTSUITE     F:/github/gdlgit/testsuite/banner_for_testsuite.pro
INCREMENT_ERRORS         F:/github/gdlgit/testsuite/test_obj_isa.pro
TEST_OBJ_ISA             F:/github/gdlgit/testsuite/test_obj_isa.pro
TEST_OBJ_ISA_FOR_LIST    F:/github/gdlgit/testsuite/test_obj_isa.pro
TEST_OBJ_ISA_FOR_OBJARR  F:/github/gdlgit/testsuite/test_obj_isa.pro

Compiled Functions:
GDL_IDL_FL               F:/github/gdlgit/testsuite/gdl_idl_fl.pro
ISA_IDL                  F:/github/gdlgit/testsuite/banner_for_testsuite.pro
ISA_INTERNAL             F:/github/gdlgit/testsuite/banner_for_testsuite.pro
PATH_SEP                 D:/SVN/gdlwc/src/pro/path_sep.pro
GDL>`

**Comment by slayoo** [comment]
@maynardGK the fix is still not in, right?

**Comment by maynardGK** [comment]
test_obj_isa should work wherever it is tested, I just ran "gdlgit" and it works here.
also, test_obj_new is not a fault of obj_new(), but of the long procedure it won't compile (over 2000 structs are manually initialized in gdlffdicom__dictionary.pro) it works ok in linux.

It is test-list and test_hash that are still lacking the full range of insertion/extraction capabiilties, some of which are touched on in the test_list procedure for ~isgit case.  
Also a bug in list is simple to find, but not  a quick fix would need a tear-down of "zombie code".

I'm working on getting my head wrapped around these two again.  As for objects in general, though,
I've made only few changes and no more planned since IDL_CONTAINER() seems completed.
